.projenrc.ts

```diff
@@ -725,6 +725,8 @@ tmpToolkitHelpers.eslint?.addRules({
   '@typescript-eslint/consistent-type-imports': 'error',
 });
 
+tmpToolkitHelpers.gitignore.addPatterns('test/**/*.map');
```

Comment:
No idea how this file got there in the first place. But it won't anymore in future.